frontend/src/apis/infomed.tsx

```diff
@@ -133,10 +133,23 @@ export const fetchHospitalInventory  = async (hospitalId: string) => {
     total: 4,
     // available: 63,
     unit: 'MT'
-    
-  }, {
-    resourceType: "Handwash",
-    total: 4,
-    available: 65
   }];
+}
+
+export interface HospitalAddress {
+  street: string;
+  city: string;
+  zip: number;
+  country: string;
+}
+
+export const fetchHospitalAddress = async (hospitalId: string) => {
+  return [
+    {
+      street: '10 Delhi House Mumbai India',
+      city: 'dehli',
+      zip: 123456,
+      country: ''
+    }
+  ]
```

Comment:
One thing I don't understand here: why is this an array? Is the hospital expected to have multiple addresses?

Reply:
(Note that it doesn't make too much sense especially that you refer only to the first element anyway later in the code.)